mooncake-store/tests/master_service_test.cpp

```diff
@@ -999,6 +999,60 @@ TEST_F(MasterServiceTest, UnmountSegmentImmediateCleanup) {
               segment2.name);
 }
 
+TEST_F(MasterServiceTest, ReadableAfterPartialUnmountWithReplication) {
+    std::unique_ptr<MasterService> service_(new MasterService(false));
+
+    // TODO: mount two larger segments when replication affinity fixed
+    // Mount two segments sized to fit exactly one replica each
+    constexpr size_t buffer1 = 0x300000000;
+    constexpr size_t buffer2 = 0x400000000;
+    constexpr size_t segment_size = 1024 * 1024 * 16;
+    constexpr size_t object_size =
+        1024 * 1024 * 16 - 16;  // force at most 1 replica per segment
```

Comment:
Nit: We’d better set object_size to a smaller value, e.g., `segment_size / 2 + 16`. There’s no guarantee that the memory allocator can allocate an object whose size is close to its capacity. If we use `1024 * 1024 * 16 - 16`, and the memory allocator’s algorithm changes in the future, this test might fail.

Reply:
fixed